packages/testcontainers/src/generic-container/abstract-started-container.ts

```diff
@@ -98,4 +98,8 @@ export class AbstractStartedContainer implements StartedTestContainer {
   public logs(opts?: { since?: number; tail?: number }): Promise<Readable> {
     return this.startedTestContainer.logs(opts);
   }
+
+  async [Symbol.asyncDispose]() {
+    await this.startedTestContainer[Symbol.asyncDispose]();
+  }
```

Comment:
@cristianrgreco Shouldn't this call `this.stop`?

```suggestion
  async [Symbol.asyncDispose]() {
    await this.stop();
  }
```